drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua

```diff
@@ -0,0 +1,81 @@
+-- Copyright 2023 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local clusters  = require "st.zigbee.zcl.clusters"
+local device_management = require "st.zigbee.device_management"
+local log = require "log"
+local Level = clusters.Level
+local OnOff = clusters.OnOff
+local ColorControl = clusters.ColorControl
+
+local emit_pushed_event = function(button_name, device)
+  local additional_fields = {
+    state_change = true
+  }
+  local event = capabilities.button.button.pushed(additional_fields)
+  local comp = device.profile.components[button_name]
+  if comp ~= nil then
+    device:emit_component_event(comp, event)
+  else
+    log.warn("Attempted to emit button event for unknown button: " .. button_name)
+  end
+end
+
+
```

Comment:
I believe this is extra whitespace as well that needs to be removed, please review.

Reply:
Remove the extra line